package.json

```diff
@@ -25,7 +25,7 @@
     "eslint-plugin-jsdoc": "^30.5.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.4",
-    "husky": "^4.3.0",
+    "husky": "6.0.0",
```

Comment:
Please use `^` to get bug fixes and new features

Reply:
I'll update it, but there are no other releases to 6. I believe it was  a temporary one that works without the engine strict of Node 12 v7

Reply:
I've updated the range